### PR TITLE
Fix jobDetails: show progress for bullmq

### PR DESCRIPTION
**Pull request by roggervalf**
#### Changes Made
This PR allows to see progress for bullmq, at the pass it only shows progress for bull queue only, because in bullmq was showing undefined 
#### Potential Risks
No potential risk
#### Test Plan
Please run the w examples:
1. npm run start:bull and open the job, after it completes the progress bar should be increased
2. npm run start:bullmq and open the job, after it completes the progress bar should be increased

#### Checklist

- [ ] I've increased test coverage
- [x] Since this is a public repository, I've checked I'm not publishing private data in the code, commit comments, or this PR.

### CODE REVIEW

**Comment by roggervalf** [comment]
@bradvogel please when you get a chance, give some :eyes: on this PR

File: example/bee.js [hunk 1]
```diff
@@ -20,8 +20,9 @@ async function main() {
   });
```
Comment (bradvogel) [inline example/bee.js:21]: I'm not able to get this to work.

When testing with `npm run start:bee` it shows 0% once the job is active:
image

When testing with `npm run start:bull` it shows undefined for progress in the active job:
image

The BullMQ example is the only one that works correctly:
image

**Review by bradvogel — CHANGES_REQUESTED** [review]

File: example/bee.js (hunk 1 above)
Comment (roggervalf) [reply example/bee.js:21]: hi @bradvogel, for bee, it is updated in succeeded state:
Selection_030
For bull I found the issue, let me address it

**Comment by roggervalf** [comment]
now for bull should appear:
Selection_031

**Comment by roggervalf** [comment]
> now for bull should appear:
> Selection_031

Please @bradvogel when you get a chance for some 👀

File: src/server/views/partials/dashboard/jobDetails.hbs [hunk 2]
```diff
@@ -75,8 +75,8 @@
   </div>
 </div>
 
-{{#unless this.queue.IS_BEE}}
 <h5>Progress</h5>
```
Comment (bradvogel) [inline src/server/views/partials/dashboard/jobDetails.hbs:78]: Are you sure "Progress" should now show up for bee queue?

**Review by bradvogel — CHANGES_REQUESTED** [review]

File: src/server/views/partials/dashboard/jobDetails.hbs (hunk 2 above)
Comment (roggervalf) [reply src/server/views/partials/dashboard/jobDetails.hbs:78]: I'm not 100% aware how the progress is used in Bee, but I can just fix the progress for bullmq for now. Let me address it

**Comment by bradvogel** [comment]
This looks fine, though the PR description doesn't match the implementation. Is the intent to ONLY show progress for Bull, and not show it for Bee and BullMQ?

**Comment by roggervalf** [comment]
> This looks fine, though the PR description doesn't match the implementation. Is the intent to ONLY show progress for Bull, and not show it for Bee and BullMQ?

oh, I just put bullmq in the description because currently we could see the progress for bull, but when running bullmq example, it is showing undefined. Now it should show bullmq progress well

**Review by bradvogel — APPROVED** [review]
oh, I see now.

**Pull request merged**